### ISSUE TITLE
Permission denied creating folder during installation

### ISSUE THREAD

**Issue by Kaw5600** [BurntSushi/nflgame#135]
error: could not create '/usr/local/lib/python2.7/dist-packages/nflgame': Permission denied

thats the error i get after the "pip install nflgame" command. Not sure what to do ? please help

**Comment by ochawkeye** [comment]
Are you able to install anything with `pip`?
Are you able to `sudo pip install nflgame`?

**Comment by waylon999** [comment]
Are you using a virtual environment?  I would recommend that over using sudo to install libraries.

**Comment by Kaw5600** [comment]
yea, im on a virtual environment

**Comment by waylon999** [comment]
Interesting. I would assume that, by looking at the error you posted, it's trying to install that into your system python instead of one in a virtualenv, so something seems off there to me. 

can you try this in a clean folder:

```
virtualenv env
. env/bin/activate
which python
```
That should show the python in your env, not the system python. If that works, try
```
pip install nflgame
```
again.

**Comment by andr3w321** [comment]
Have you tried `pip install nflgame --user`

**Comment by Kaw5600** [comment]
thanks you guys so much for your help. .it seems "pip install nflgame  --user" worked

**Comment by Kaw5600** [comment]
ok, so i got that to work but now in Python, when i enter "import nflgame" i get this error:

![nflgameerror](https://cloud.githubusercontent.com/assets/14000966/9824580/c0864eee-589b-11e5-80e1-c95c6fe4aef7.PNG)

**Comment by ochawkeye** [comment]
![image](https://cloud.githubusercontent.com/assets/2996205/9824676/fd146ec0-5893-11e5-90e1-680f955daf76.png)

`nflgame` is Python 2.x only at this point.

**Comment by Kaw5600** [comment]
gotcha, thanks!!

**Comment by Kaw5600** [comment]
in the python command line of the python 2.7 i installed, i still get:
![nflgameerror](https://cloud.githubusercontent.com/assets/14000966/9825296/4a5c1848-58a0-11e5-913d-85ff3a2ad020.PNG)

**Comment by BurntSushi** [comment]
Please provide a complete list of commands and their outputs that you ran in order to install nflgame.

**Comment by Kaw5600** [comment]
i followed the tutorial called "Tutorial for non programmers: Installation and examples" and followed the installation steps for the microsoft dl where no commands were given. the only command i EVER used was "pip install nflgame --user". that was in putty where it seemed to have installed successfully but when i try the command import "nflgame" in Python 2.7.10 (IDLE) i get the error message above.

**Comment by ochawkeye** [comment]
> that was in putty

Were you remotely accessing a system that you installed `nflgame` to when you were in putty?  
I ask because there are a couple of path names referenced in this thread:

`'/usr/local/lib/python2.7/dist-packages/nflgame':` : This is not a Windows path...Is this where you remote installed `nflgame` via Putty?
`C:\Python34\lib\site-packages\nflgame`: This is a Windows path, but it is for Python 3.4

`C:\Python27\python.exe` doesn't have visibility to a package installed in the `C:\Python34\` lib folder and it doesn't have visibility to a package installed on a remote system.

If you want to `import nflgame` on your local Windows machine, you're going to need to get `nflgame` installed into your `C:\Python27\Lib\site-packages\` folder.  

How did you get it installed into `C:\Python34`?  Was that via `pip`?  If so, do you have the 2.7 version of `pip` as well?  If so, can you try running `C:\Python27\Scripts\pip.exe install nflgame`?

**Comment by Kaw5600** [comment]
yes, i got it installed into C:\Python34 via pip and "C:\Python27\Scripts\pip.exe install nflgame" worked perfectly. Thanks alot !
